example/molbe_h8_chemical_potential.py

```diff
@@ -33,7 +33,7 @@
 # Perform BE calculations with different fragment schemes:
 
 # Define BE1 fragments
-fobj = fragpart(be_type="be1", mol=mol)
+fobj = fragpart(be_type="be1", frag_type="autogen", mol=mol)
```

Comment:
`graphgen` can do BE1 as well, right? Do you want to add a block (like be2 and be3 below) that demonstrates `graphgen` for be1, too?

Reply:
Sure, don't you also want to show that BE1 can be done with graphgen in this example file for consistency, though? Unresolving this